### PR TITLE
feat: add run_jobs to aws base embedding

### PR DESCRIPTION
**Pull request by EmanuelCampos**

### CODE REVIEW

File: llama-index-core/llama_index/core/base/embeddings/base.py [hunk 1]
```diff
@@ -347,6 +352,7 @@ async def aget_text_embedding_batch(
     ) -> List[Embedding]:
         """Asynchronously get a list of text embeddings, with batching."""
         dispatch_event = dispatcher.get_dispatch_event()
+        num_workers = self.num_workers
```
Comment (sourabhdesai) [inline llama-index-core/llama_index/core/base/embeddings/base.py:355]: Should `num_workers` just be a method argument of `aget_text_embedding_batch`?

Comment (EmanuelCampos) [reply llama-index-core/llama_index/core/base/embeddings/base.py:355]: we would need to put on __call__ as well, to work with transformations

**Comment by logan-markewich** [comment]
I realized this change depends on a bump in core -- will need to manually publish the core changes first, then change the versions for the embeddings.

We can merge this now without changing the toml versions, and I can handle the publishing sequence today

File: llama-index-core/llama_index/core/base/embeddings/base.py (hunk 1 above)
Comment (logan-markewich) [reply llama-index-core/llama_index/core/base/embeddings/base.py:355]: I think its in line with the current interface (batch size is also a class attribute)

**Review by logan-markewich — APPROVED** [review]

**Pull request merged**